src/cli/main.rs

```diff
@@ -35,6 +35,7 @@ fn _main() -> Result<(), i32> {
 
     let io = emulator::IoRegisters::new();
     let mut computer = Computer::new(&io);
+    let mut last_file = String::from("none");
```

Comment:
Type `Option<String>` would be more idiomatic

Reply:
I am really not into rust at all, needa how `Option<String>`  works